<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QVocabulary</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <widget class="QMenu" name="menu_Open_Database_Recent">
     <property name="title">
      <string>&amp;Open Database Recent</string>
     </property>
     <addaction name="actionEmpty"/>
    </widget>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionImportMicrosoftVocabulary"/>
    </widget>
    <addaction name="actionNew_DB"/>
    <addaction name="actionOpen_DB"/>
    <addaction name="menu_Open_Database_Recent"/>
    <addaction name="separator"/>
    <addaction name="menuImport"/>
    <addaction name="separator"/>
    <addaction name="actionInsertGroup"/>
    <addaction name="actionInsertWord"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuDatabase"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_DB"/>
   <addaction name="actionOpen_DB"/>
   <addaction name="actionInsertGroup"/>
   <addaction name="actionInsertWord"/>
  </widget>
  <action name="actionOpen_DB">
   <property name="text">
    <string>&amp;Open Database</string>
   </property>
   <property name="toolTip">
    <string>Open Database</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionInsertWord">
   <property name="text">
    <string>Insert &amp;Word</string>
   </property>
   <property name="toolTip">
    <string>Insert a Word</string>
   </property>
  </action>
  <action name="actionNew_DB">
   <property name="text">
    <string>&amp;New Database</string>
   </property>
   <property name="toolTip">
    <string>New Database</string>
   </property>
  </action>
  <action name="actionInsertGroup">
   <property name="text">
    <string>&amp;Insert Group</string>
   </property>
   <property name="toolTip">
    <string>Insert a Group</string>
   </property>
  </action>
  <action name="actionEmpty">
   <property name="text">
    <string>Empty</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionImportMicrosoftVocabulary">
   <property name="text">
    <string>Import From Microsoft Vocabulary++ (*.xml)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
